crates/router/src/utils/user/sample_data.rs

```diff
@@ -374,7 +388,35 @@ pub async fn generate_sample_data(
             None
         };
 
-        res.push((payment_intent, payment_attempt, refund));
+        // TODO: change dummy dispute data to random dispute data
```

Comment:
We can remove this dummy data, let's cover this as well in next PR. 
It's good that you tested with some dummy data. But let's keep this PR to extending sample data interface.